<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-7VQ11PQH" ignored="11" name="Default test" tests="1" failures="1" timestamp="2021-02-08T13:29:29 IST" time="0.068" errors="0">
  <testcase name="close" time="0.011" classname="xendit.xendit_qa_assesment.addition">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at xendit.xendit_qa_assesment.addition.close(addition.java:113)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- close -->
  <testcase name="TC3" time="0.0" classname="xendit.xendit_qa_assesment.addition">
    <ignored/>
  </testcase> <!-- TC3 -->
  <testcase name="TC1Compare" time="0.0" classname="xendit.xendit_qa_assesment.addition">
    <ignored/>
  </testcase> <!-- TC1Compare -->
  <testcase name="TC4Compare" time="0.0" classname="xendit.xendit_qa_assesment.addition">
    <ignored/>
  </testcase> <!-- TC4Compare -->
  <testcase name="openOnlineCalculator" time="0.0" classname="xendit.xendit_qa_assesment.addition">
    <ignored/>
  </testcase> <!-- openOnlineCalculator -->
  <testcase name="TC5Compare" time="0.0" classname="xendit.xendit_qa_assesment.addition">
    <ignored/>
  </testcase> <!-- TC5Compare -->
  <testcase name="TC3Compare" time="0.0" classname="xendit.xendit_qa_assesment.addition">
    <ignored/>
  </testcase> <!-- TC3Compare -->
  <testcase name="TC2Compare" time="0.0" classname="xendit.xendit_qa_assesment.addition">
    <ignored/>
  </testcase> <!-- TC2Compare -->
  <testcase name="TC2" time="0.0" classname="xendit.xendit_qa_assesment.addition">
    <ignored/>
  </testcase> <!-- TC2 -->
  <testcase name="TC1" time="0.0" classname="xendit.xendit_qa_assesment.addition">
    <ignored/>
  </testcase> <!-- TC1 -->
  <testcase name="TC4" time="0.0" classname="xendit.xendit_qa_assesment.addition">
    <ignored/>
  </testcase> <!-- TC4 -->
  <testcase name="TC5" time="0.0" classname="xendit.xendit_qa_assesment.addition">
    <ignored/>
  </testcase> <!-- TC5 -->
</testsuite> <!-- Default test -->
